### PR TITLE
[Bug Fix] SSH 1.1.1

### PR DESCRIPTION
**Pull request by vthiebaut10**
- Version 1.1.0 introduces a regression that was affecting multiple customers. It changed that way that the extension looks for the OpenSSH client executables, from looking on the C:\Windows\System32\OpenSSH folder to relying on the PATH environment variable being set properly. However, for many costumers the extension wasn't finding the executables, even when the path was correct. We reverted the change.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
SSH

**Review by yonzhan — APPROVED** [review]

File: src/ssh/azext_ssh/_params.py [hunk 1]
```diff
@@ -26,8 +26,8 @@ def load_arguments(self, _):
                    help='Folder path that contains ssh executables (ssh.exe, ssh-keygen.exe, etc). '
                    'Default to ssh pre-installed if not provided.')
         c.argument('delete_credentials', options_list=['--force-delete-credentials', '--delete-private-key'],
-                   help=('This is an internal argument. This argument is used by Azure Portal to provide a one click '
-                         'SSH login experience in Cloud shell.'),
+                   help=('[Deprecated] This is an internal argument. This argument is used by Azure Portal to '
```
Comment (jiasli) [inline src/ssh/azext_ssh/_params.py:29]: Deprecation should use the `deprecate_info` convention, such as

https://github.com/Azure/azure-cli/blob/96bbce449c51c829d6f4d2518fafd5a46c0314e8/src/azure-cli/azure/cli/command_modules/storage/_params.py#L780

```py
        c.argument('socket_timeout', deprecate_info=c.deprecate(hide=True),
                   help='The socket timeout(secs), used by the service to regulate data flow.')
```

**Comment by jiasli** [comment]
> Version 1.1.0 introduces a regression that was affecting multiple customers.

Please kindly specify what the regression is.

File: src/ssh/azext_ssh/_params.py (hunk 1 above)
Comment (vthiebaut10) [reply src/ssh/azext_ssh/_params.py]: @jiasli The deprecated_info is set on line 31

**Review by jiasli — APPROVED** [review]

**Pull request merged**